iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AbstractCreateConnectableFeeders.java

```diff
@@ -0,0 +1,153 @@
+        for (int side : sides) {
+            String bbsId = getBbsId(side);
+            BusbarSection bbs = network.getBusbarSection(bbsId); //If the busbar exists, topology of the associated voltage level is node/breaker
+            if (bbs == null) {
+                LOGGER.error("Busbar section {} not found.", bbsId);
+                notFoundBusbarSectionReport(reporter, bbsId);
+                if (throwException) {
+                    throw new PowsyblException(String.format("Busbar section %s not found.", bbsId));
+                }
+                return;
+            }
+            VoltageLevel voltageLevel = bbs.getTerminal().getVoltageLevel();
+            int connectableNode = voltageLevel.getNodeBreakerView().getMaximumNodeIndex() + 1;
+            setNode(side, connectableNode, voltageLevel.getId());
+        }
+
+        Connectable<?> connectable = add();
+        if (connectable.getNetwork() != network) {
+            connectable.remove();
+            LOGGER.error("Network given in parameters and in connectableAdder are different. Connectable '{}' of type {} was added then removed",
+                    connectable.getId(), connectable.getType());
+            networkMismatchReport(reporter, connectable.getId(), connectable.getType());
+            if (throwException) {
+                throw new PowsyblException("Network given in parameters and in connectableAdder are different. Connectable was added then removed");
+            }
+            return;
+        }
+        String connectableId = connectable.getId();
+
+        boolean createConnectablePosition = false;
+        ConnectablePositionAdder<?> connectablePositionAdder = connectable.newExtension(ConnectablePositionAdder.class);
+        for (int side : sides) {
+            VoltageLevel voltageLevel = getVoltageLevel(side, connectable);
+            Set<Integer> takenFeederPositions = TopologyModificationUtils.getFeederPositions(voltageLevel);
+            int positionOrder = getPositionOrder(side);
+            if (!takenFeederPositions.isEmpty()) {
+                if (takenFeederPositions.contains(positionOrder)) {
+                    LOGGER.error("PositionOrder {} already taken.", positionOrder);
+                    positionOrderAlreadyTakenReport(reporter, positionOrder);
+                    if (throwException) {
+                        throw new PowsyblException(String.format("PositionOrder %d already taken.", positionOrder));
+                    }
+                    return;
+                }
+                getFeederAdder(side, connectablePositionAdder)
+                        .withDirection(getDirection(side))
+                        .withOrder(positionOrder)
+                        .withName(connectableId)
+                        .add();
+                createConnectablePosition = true;
+            } else {
+                LOGGER.warn("No extensions found on voltageLevel {}. The extension is not created.", voltageLevel.getId());
```

Comment:
In fact it means two things here:
- either there are feeders but no extensions on them, then not adding the extension might be ok (I would add it anyway, but that was already discussed)
- or there are no feeders hence no extensions; for instance you're creating a voltage level from scratch and you first add the busbar, then the line(s), then the other feeders. In that case we should add the extension anyway, don't you think so? To that purpose, we could just check that `voltageLevel.getConnectables()` is empty.

Reply:
I followed your note, only checking if getConnectables() contains one and only one element (the connectable we've just created)